README.md

```diff
@@ -233,6 +233,22 @@ const banjarCollection = [
 ];
 
 const materialCollection = [
+   {
+	worker: {
```

Comment:
what does `worker` represent in this instance? we'll have to `i18n` the database modals

Reply:
It is from 04.0 screen (also employee = true in person model), should we call it employee?